docs/CMakeLists.txt

```diff
@@ -30,15 +30,15 @@ add_custom_target(${PROJECT_NAME}_docs
       BUILD_DIR=${CMAKE_CURRENT_BINARY_DIR} ${SPHINX_EXECUTABLE} ${SPHINX_HTML_ARGS} ${SPHINX_SOURCE} ${SPHINX_BUILD}
    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
    COMMENT "Generating documentation with Sphinx"
-   DEPENDS morpheus-package-outputs
+   DEPENDS morpheus-package-outputs morpheus_llm-package-outputs
 )
 
 add_custom_target(${PROJECT_NAME}_docs_linkcheck
    COMMAND
       BUILD_DIR=${CMAKE_CURRENT_BINARY_DIR} ${SPHINX_EXECUTABLE} ${SPHINX_LINKCHECK_ARGS} ${SPHINX_SOURCE} ${SPHINX_LINKCHECK_OUT}
    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
    COMMENT "Checking documentation links with Sphinx"
-   DEPENDS morpheus-package-outputs
+   DEPENDS morpheus-package-outputs morpheus_llm-package-outputs
```

Comment:
The problem is that `MORPHEUS_BUILD_MORPHEUS_LLM` defaults to `OFF`, if you build with `MORPHEUS_BUILD_MORPHEUS_LLM=OFF` then you get an error:
```
ninja: error: 'docs/morpheus_llm-package-outputs', needed by 'docs/CMakeFiles/morpheus_docs', missing and no known rule to make it
```

Reply:
good catch. I missed that. I will make the llm dependency conditional.